fdm-calculator/src/doses/get-dose-field.ts

```
@@ -0,0 +1,43 @@
+        const farmId = field.b_id_farm
+
+        // Get the properties of the fertilizers that are used for the applications
+        const fertilizers = await getFertilizers(fdm, farmId)
+
+        // Calculate the dose per nutrient for this field
+        return calculateDose({ applications, fertilizers })
+    } catch (error) {
+        throw new Error(`Failed to calculate dose for field ${b_id}: ${error.message}`)
+    }
+
+    return doses
```

Biome (1.9.4)

[error] This code is unreachable

... because either this statement will return from the function ...

... or this statement will throw an exception beforehand


(lint/correctness/noUnreachable)